src/polytopes/ring.jl

```diff
@@ -58,17 +58,16 @@ Base.open(r::Ring) = open(Rope(parent(r.vertices)))
 # do not change which vertex comes first for closed chains
 Base.reverse!(r::Ring) = (reverse!(@view r.vertices[(begin + 1):end]); r)
 
+# TODO: check dim: 2
```

Comment:
no need to check this, we may add a specialized algorithm in the future, so in theory it could work in 3D